### PR TITLE
Create v.2.40.0.md

### PR DESCRIPTION
**Pull request by ViktarStarastsenka**
please do not merge it for now

### CODE REVIEW

**Comment by ViktarStarastsenka** [comment]
@rrelledge , we are ready to release them when you can. Thanks!

File: content/ri/release-notes/v.2.40.0.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- [#2879](https://github.com/RedisInsight/RedisInsight/pull/2879) UX improvements to simplify in-app provisioning of a free [Redis Cloud](https://redis.com/comparisons/oss-vs-enterprise/?utm_source=redisinsight&utm_medium=rel_notes&utm_campaign=2_40) database. Create a new database with a preselected cloud vendor and region by using the recommended sign-up settings. You can manage your database by signing in to the [Redis Cloud console](https://app.redislabs.com/#/databases?utm_source=redisinsight&utm_medium=rel_notes&utm_campaign=2_40)
+- [#2851](https://github.com/RedisInsight/RedisInsight/pull/2851) See plan, cloud vendor, and region details after successfully provisioning your free [Redis Cloud](https://redis.com/comparisons/oss-vs-enterprise/?utm_source=redisinsight&utm_medium=rel_notes&utm_campaign=2_40) database
+- [#2882](https://github.com/RedisInsight/RedisInsight/pull/2882) Optimizations when uploading large text files with the list of Redis commands, available under bulk actions in Browser
+- [#2808](https://github.com/RedisInsight/RedisInsight/pull/2808) Enhanced security measurement to no longer display in plain text existing passwords for Redis Sentinel
```
Comment (rrelledge) [inline content/ri/release-notes/v.2.40.0.md:23]:
```suggestion
- [#2808](https://github.com/RedisInsight/RedisInsight/pull/2808) Enhanced security measurement to no longer display existing passwords for Redis Sentinel in plain text
```

File: content/ri/release-notes/v.2.40.0.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+This is the General Availability (GA) release of RedisInsight 2.40.
+
+### Highlights
+- To simplify in-app provisioning of a free [Redis Cloud](https://redis.com/comparisons/oss-vs-enterprise/?utm_source=redisinsight&utm_medium=rel_notes&utm_campaign=2_40) database to use with RedisInsight interactive tutorials, the recommended cloud vendor and region are now preselected 
+- Optimizations when uploading large text files with the list of Redis commands, available under bulk actions in Browser
+
+### Details
+
+**Features and improvements**
+- [#2879](https://github.com/RedisInsight/RedisInsight/pull/2879) UX improvements to simplify in-app provisioning of a free [Redis Cloud](https://redis.com/comparisons/oss-vs-enterprise/?utm_source=redisinsight&utm_medium=rel_notes&utm_campaign=2_40) database. Create a new database with a preselected cloud vendor and region by using the recommended sign-up settings. You can manage your database by signing in to the [Redis Cloud console](https://app.redislabs.com/#/databases?utm_source=redisinsight&utm_medium=rel_notes&utm_campaign=2_40)
```
Comment (rrelledge) [inline content/ri/release-notes/v.2.40.0.md:20]:
```suggestion
- [#2879](https://github.com/RedisInsight/RedisInsight/pull/2879) UX improvements to simplify in-app provisioning of a free [Redis Cloud](https://redis.com/comparisons/oss-vs-enterprise/?utm_source=redisinsight&utm_medium=rel_notes&utm_campaign=2_40) database. Create a new database with a preselected cloud vendor and region by using the recommended sign-up settings. You can manage your database by signing in to the [Redis Cloud console](https://app.redislabs.com/#/databases?utm_source=redisinsight&utm_medium=rel_notes&utm_campaign=2_40).
```

**Review by rrelledge — APPROVED** [review]
@ViktarStarastsenka Looks good to me. I just left two small suggestions. Please let me know when you are ready for me to merge and publish this PR.

**Comment by ViktarStarastsenka** [comment]
@rrelledge , thank you!
added changes, please merge it

**Pull request merged**